### PR TITLE
Corrected controversial message in README

### PR DESCRIPTION
**Pull request by barvhaim**

### CODE REVIEW

**Review by yuvalapidot — APPROVED** [review]

**Comment by ranst91** [comment]
You may aswell remove the other one. Political message or personal agenda has no room in software documentation, regardless if it's right or wrong.
If GraphiQL devs are bothered, they are more than welcome to add these messages as a snackbar/other UI option in their official website.
Having these things contained inside one's code is purely wrong.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!-- @format -->
-> # Black Lives Matter 🖤 Free Palestine 🇵🇸
+> # Black Lives Matter 🖤
```
Comment (ranst91) [inline README.md:2]:
```suggestion

```

**Review by ranst91 — APPROVED** [review]
You may aswell remove the other one. Political message or personal agenda has no room in software documentation, regardless if it's right or wrong.
If GraphiQL devs are bothered, they are more than welcome to add these messages as a snackbar/other UI option in their official website.
Having these things contained inside one's code is purely wrong.

**Review by yardenshoham — APPROVED** [review]

**Review by zivchen — APPROVED** [review]

**Comment by leebyron** [comment]
The GraphQL technical steering committee is working on a policy to guide the use of political speech from the voice of a GraphQL project. The original author chose to revert the original change via #1879 until we complete that policy. This should resolve this issue.

As a reminder, all discussion within GraphQL projects must adhere to our [Code of Conduct](https://github.com/graphql/foundation/blob/main/CODE-OF-CONDUCT.md).

**Pull request closed without merging**